### PR TITLE
Update guide to use crates.io dependencies

### LINKED ISSUE #1930: Getting Started shouldn't show git deps first
http://doc.crates.io/guide.html#adding-dependencies currently shows a git dep, but it should show a crates.io dep first

### PR DESCRIPTION
**Pull request by steveklabnik**
Fixes #1930

### CODE REVIEW

File: src/doc/guide.md [hunk 1]
```diff
@@ -136,68 +125,74 @@ project.
...
-]
-
-[[package]]
-name = "color"
-version = "0.1.0"
-source = "git+https://github.com/bjz/color-rs.git#bf739419e2d31050615c1ba1a395b474269a4b98"
-
-```
+Our `Cargo.lock` contains the exact information about which revision of all of
+these dependencies we used:
```
Comment (huonw) [inline src/doc/guide.md]: Should there be code after this : ?

**Comment by huonw** [comment]
There seems to be more instances of git dependencies below.

File: src/doc/guide.md (hunk 1 above)
Comment (steveklabnik) [inline src/doc/guide.md]: Ahh, so, I decided to remove the actual `toml` code, as we don't actually specify the format of it, and it's also much less illustrative than the `git` example. But, it shouldn't have a `:` at the end, but a `.`

**Pull request merged**